modules/apps/headless/headless-space/headless-space-impl/rest-openapi.yaml

```diff
@@ -166,4 +166,59 @@ paths:
                         application/xml:
                             schema:
                                 $ref: "#/components/schemas/Space"
+            tags: ["Space"]
+    /spaces/{spaceId}/link/{toSiteId}:
```

Comment:
"link" should be a collection, so it should be "links" or "linked-sites" or something like that, not a singular name.

Reply:
I've agreed with chete about this.